app/src/main/res/layout/activity_home.xml

```diff
@@ -4,152 +4,226 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".Home">
+    tools:context=".Home"
+    tools:layout_editor_absoluteY="81dp">
 
     <SearchView
-        android:id="@+id/searchView2"
-        android:layout_width="52dp"
+        android:id="@+id/search_Recordingbutton"
```

Comment:
Choose one or the other, either camelCase or snake_case
Every word after the first should be capitalized so searchRecordingButton

Reply:
Its the easiest to change in the design XML at the top right where it says ID

Reply:
fixed